.github/workflows/migration-sql.yml

```diff
@@ -0,0 +1,84 @@
+name: Print alembic migrations SQL
+
+on:
+  pull_request:
+    branches: [master, sql-actions]
+
+jobs:
+  print_sql_upgrades:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Won't this check always succeed in case of a PR?

Reply:
Removed.